Detections/AzureActivity/RareOperations.yaml

```diff
@@ -0,0 +1,33 @@
+id: 23de46ea-c425-4a77-b456-511ae4855d69
+name: Rare subscription-level operations in Azure
+description: |
+  'This query looks for a few sensitive subscription-level events based on Azure Activity Logs. 
+   For example this monitors for the operation name 'Create or Update Snapshot' which is used for creating backups but could be misused by attackers 
+   to dump hashes or extract sensitive information from the disk.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Credential Access 
+  - Persistence, 
+relevantTechniques:
+  - T1003
+  - T1098
+
+query: |
+ 
+  let starttime =1d;
+  let endtime = 1h;
+  let SensitiveOperationList = dynamic(
+  ["List keys", "List Storage Account Keys", "Register Subscription", "Create or Update Snapshot", "Create or Update Network Security Group"]);
+  AzureActivity
+  | where TimeGenerated between (ago(starttime)..ago(endtime))
+  | where ActivityStatus == "Succeeded"
```

Comment:
Recommend =~ to avoid missing due to case.

Reply:
Thanks. Done.